### PR TITLE
Write operations

### PR DESCRIPTION
**Pull request by kvikshaug**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -31,7 +31,7 @@ universal = 1
 universal = 1
 
 [flake8]
-ignore = D100,D101,D102,D103,D104,D105,D106,D107
+ignore = D100,D101,D102,D103,D104,D105,D106,D107,W504
```
Comment (Midnighter) [inline setup.cfg]: I'd prefer if that was ignored on a per line basis.

File: src/map_storage/jwt.py [hunk 2]
```diff
@@ -45,7 +44,7 @@ def decode_jwt():
         try:
             _, token = auth.split(' ', 1)
             g.jwt_claims = jwt.decode(
-                token, app.config['JWT_PUBLIC_KEY'], app.config['ALGORITHM'])
+                token, app.config['JWT_PUBLIC_KEY'], 'RS512')
```
Comment (Midnighter) [inline src/map_storage/jwt.py]: How come the change?

**Review by Midnighter — APPROVED** [review]
:+1:

File: src/map_storage/jwt.py (hunk 2 above)
Comment (kvikshaug) [reply src/map_storage/jwt.py]: This was left over from the IAM service which is the service that defines the algorithm in the settings. It should rather be extracted from the public key here I suppose, I'll append a fix for that.

File: setup.cfg (hunk 1 above)
Comment (kvikshaug) [reply setup.cfg]: Ok. Curious how black would split up those lines.

**Pull request merged**